bpm/bonita-core/bonita-process-instance/bonita-process-instance-impl/src/main/resources/org/bonitasoft/engine/core/process/instance/model/impl/hibernate/process.instance.queries.hbm.xml

```diff
@@ -83,6 +83,7 @@
 		WHERE g.logicalGroup4 = :parentProcessInstanceId
 		AND g.name = :name
 		AND g.terminal = FALSE
+		AND g.hitBys NOT LIKE 'FINISH%'
```

Comment:
I'am afraid problems will come with volume. In case a few rows, index/table scan is quite the same due to object cache.